core/src/main/java/com/linecorp/armeria/client/ClientFactoryOptions.java

```diff
@@ -526,4 +542,13 @@ public Http1HeaderNaming http1HeaderNaming() {
     public boolean tlsAllowUnsafeCiphers() {
         return get(TLS_ALLOW_UNSAFE_CIPHERS);
     }
+
+    /**
+     * Returns the {@link ChannelPipeline} customizer set. This consumer is no-op
+     * by default.
```

Comment:
Could you copy the description in `#CHANNEL_PIPELINE_CUSTOMIZER`?

Reply:
👍